### ISSUE TITLE
journal title and series name are duplicated

### ISSUE THREAD

**Issue by wujastyk** [michel-kraemer/citeproc-java#111]
This is an issue that has arisen in the context of using this library in JabRef ([JabRef issue](https://github.com/JabRef/jabref/issues/8279#issue-1064757005))

### JabRef version

Latest development branch build (please note build date below)

### Operating system

GNU / Linux

### Details on version and operating system

JabRef 5.4--2021-11-25--7aaf5e2 Linux 5.11.0-40-generic amd64  Java 16.0.2  JavaFX 17.0.1+1

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

1. go to the preview tab for any article etc.
2. Display has the journal title duplicated:

Journal:

![image](https://user-images.githubusercontent.com/762246/143621958-e33161c8-fdb8-4342-b863-ffb8b120c66b.png)

Book in series:

![image](https://user-images.githubusercontent.com/762246/143622024-ce4121f9-812f-491b-9188-3c2edc7080b0.png)

I have tried a dozen or so different preview styles.  Above are "Chicago Manual of Style 17th ed" and "Oxford Studies in Ancient Philosophy", but it's the same whatever style I choose.

I wouldn't mind editing the preview template to my liking, but that's currently not available

### Appendix

```
@Article{wixt-2018,
  author           = {John T. Wixted and Laura Mickes and Ronald P. Fisher},
  date             = {2018-05},
  journal          = {Perspectives on Psychological Science},
  number           = {3},
  pages            = {324--335},
  title            = {Rethinking the Reliability of Eyewitness Memory},
  volume           = {13},
  doi              = {10.1177/1745691617734878},
  abstract         = {Although certain pockets within the broad field of academic psychology have come to appreciate that eyewitness memory is more reliable than was once believed, the prevailing view, by far, is that eyewitness memory is unreliable—a blanket assessment that increasingly pervades the legal system. On the surface, this verdict seems unavoidable: Research convincingly shows that memory is malleable, and eyewitness misidentifications are known to have played a role in most of the DNA exonerations of the innocent. However, we argue here that, like DNA evidence and other kinds of scientifically validated forensic evidence, eyewitness memory is reliable if it is not contaminated and if proper testing procedures are used. This conclusion applies to eyewitness memory broadly conceived, whether the test involves recognition (from a police lineup) or recall (during a police interview). From this perspective, eyewitness memory has been wrongfully convicted of mistakes that are better construed as having been committed by other actors in the legal system, not by the eyewitnesses themselves. Eyewitnesses typically provide reliable evidence on an initial, uncontaminated memory test, and this is true even for most of the wrongful convictions that were later reversed by DNA evidence.},
  creationdate     = {2021-11-26T11:28:38},
  modificationdate = {2021-11-26T11:58:46},
  publisher        = {{SAGE} Publications},
}

@Book{qvar-yoga2,
  author           = {Olle Qvarnström},
  date             = {2013},
  location         = {Mumbai},
  publisher        = {Hindi Granth Karyalay},
  title            = {A Handbook on the Three Jewels of Jainism: The Yogaśāstra of Hemacandra},
  edition          = {6},
  isbn             = {9780674009349},
  series           = {Harvard Oriental Series},
  comment          = {Revised edition of \cite{qvar-yoga}. Hemacandra lived in the late 11th century.},
  file             = {:Quarnström, O/Quarnström_The Yogasastra of Hemacandra_2002.pdf:PDF},
  modificationdate = {2018-09-17T00:00:00},
}


```

**Comment by ThiloteE** [comment]
I was advised to post my investigation details here as well:

The same shows up also under Jabref `options>preferences>entry preview`. Not all citationstyles do have this duplication, but many do. For some it is the `journal` or `series` fields, for some it only is the `number` field.

- I also tested an entry with Zotero and there it works fine:

   `Thompson, Mark R. ‘Middle-Class Remorse: Re-Embracing Liberal Democracy in the Philippines and Thailand’. Global Asia, vol. 15, no. 1, East Asia Foundation, Mar. 2020, pp. 60–64.`

- In Jabref we have the dublicate:

   ![image](https://user-images.githubusercontent.com/73715071/143773434-302ada6f-b872-483d-8e61-462930c31fff.png)

- Furthermore, I checked with Winmerge if the citation-style code is any different between the ones used by Zotero (https://www.zotero.org/styles/modern-language-association) and the one from Jabref (`Options>preferences>entry preview>Modern Language Association 9th edition`) and there was **no difference**, except **ONE** difference and that was, according to Winmerge, both used **different line-break format**.

   The method that allowed me to extract the citation-style code from Jabref was to go to (Options>preferences>entry preview>Modern Language Association 9th edition>edit) and copy paste it into the editor (= the text editor on Windows 10) and save it as a textfile and then compare it with the one from Zotero.

   Not sure if the line-break format is applied by saving the style in 'Editor' or if it is created by citeproc-java or rather by Jabref.

I hope that helps in solving this
